### PR TITLE
Added rendering of tag tourism=wilderness_hut

### PR DESCRIPTION
**Pull request by Ircama**
This PR squashes all commits in PR #2140, which can be closed.

**Added rendering of tag tourism=wilderness_hut**

Added specific symbols for basic_hut and wilderness_hut vs. generic amenity=shelter.

Notice that this drop shall be consistent with the other push related to zooms, where alpine_hut symbol (the main one) is shown at zoom>=13.

To summarize, the following different kind of huts have specific symbols now:

- amenity=shelter, shelter_type=basic_hut (zoom>=13)
- amenity=shelter (only shown at zoom >=16)
- tourism=alpine_hut  (not modified within this drop)
- tourism=wilderness_hut (zoom>=13)

alpine_hut: symbols/alpinehut.p.16.png
wilderness_hut: symbols/wilderness_hut.svg
generic shelter: symbols/shelter-14.svg
amenity=shelter with shelter_type=basic_hut: symbols/basic_hut.svg

### CODE REVIEW

**Comment by matkoniecz** [comment]
Note that shelter_type type tag is currently unavailable in database.

Without hstore only tags listed at https://github.com/gravitystorm/openstreetmap-carto/blob/master/openstreetmap-carto.style are available.

File: project.yaml [hunk 1]
```diff
@@ -1666,11 +1666,12 @@ Layer:
             denomination,
             "generator:source",
             power_source,
+            tags->'shelter_type'::text AS shelter_type,
```
Comment (pnorman) [inline project.yaml]: You can't use the shelter_type tag at this time

**Comment by Ircama** [comment]
> You can't use the shelter_type tag at this time

OK, I'm removing hstore usage

**Comment by math1985** [comment]
> OK, I'm removing hstore usage

@Ircama Did you remember to push your last version? It seems you're still using the shelter_type tag, which is not available in our database.

**Comment by Ircama** [comment]
> @Ircama Did you remember to push your last version? It seems you're still using the shelter_type tag, which is not available in our database.

I apologize for this delay. I had too limited network bandwidth last days, but can go on working on this PR this week

**Comment by math1985** [comment]
No problem at all! Just wanted to make sure you weren't waiting for a review.

**Comment by Ircama** [comment]
New PR to this feature with removed usage of shelter_type is at
https://github.com/gravitystorm/openstreetmap-carto/pull/2279

**Pull request closed without merging**